dmapiclient/audit.py

```diff
@@ -39,6 +39,7 @@ class AuditTypes(Enum):
     send_clarification_question = "send_clarification_question"
     send_application_question = "send_application_question"
     answer_selection_questions = "answer_selection_questions"
+    agree_framework_variation = "agree_framework_variation"
```

Comment:
Yep we should definitely have had a new audit event for that `supplier_update` I think is more like changing supplier details like address or whatever. Worth a discussion if we should change it now or not.

This audit event probably does belong in the `Framework agreements` section, even though it is a thing that suppliers do.

But :+1: either way.